.jenkins/precheckin.groovy

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env groovy
+@Library('rocJenkins@pong') _
+import com.amd.project.*
+import com.amd.docker.*
+import java.nio.file.Path;
+
+def runCI = 
+{
+    nodeDetails, jobName->
+
+    def prj = new rocProject('rocHPCG', 'PreCheckin')
+    prj.paths.build_command = './install.sh -t --with-openmp=OFF --with-mpi=OFF'
+    prj.libraryDependencies = ['rocPRIM']
```

Comment:
What does this line do?

Reply:
I actually don't think it's needed. I will remove it